### PR TITLE
:rocket: Deploy PV-T changes to dev branch

### PR DESCRIPTION
**Pull request by BenWinchester**
### Description
Solar photovoltaic (PV) panels are widely used throughout the world. However, they are not the only means of harnessing renewable energy in an off-grid environment. Novel hybrid photovoltaic-thermal (PV-T) panels are another way of doing so, and, as well as synergistic efficiency benefits, they provide a means of extracting both heat and electricity from the same area.

Modelling these collectors in CLOVER is a step towards covering the full range of useful functionality for researchers, NGOs, private companies and various other stakeholders.

This pull request merges and deploys these PV-T features to the development branch of CLOVER. The versioning is updated to `5.1.0b1` to reflect this new feature (`5.1`) and the development/beta-testing nature of the release (`b1`).

### Linked Issues
This pull request does not close or resolve any open issues.

### Unit tests
This pull request:
* modifies the integration testing for CLOVER to change the variable `unmet_energy_fraction` to `unmet_electricity_fraction` to reflect the concept of different types of energy being introduced.

## Dependencies
This pull request is conditional on the changes from PR #73 being approved and integrated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,42 @@ This will fetch the latest stable version of CLOVER and install it into your cur
 
 To download the CLOVER source code directly from Github, simply click the green `Code` button near the top of this page, and select `Download ZIP`. Once downloaded, unpack the zip file into a directory of your choice. You will now be able to run CLOVER from a terminal in this directory. Use the `cd` command to change the directory of your terminal to the extracted folder in order to run CLOVER.
 
+#### Downloading Git LFS files
```
Comment (phil-sandwell) [inline README.md:39]: Overall I think this process looks fine, especially at this stage. Does the implementation of this in the long term depend on the limitations of the Git LFS? Or somewhere else to host these files?

Comment (BenWinchester) [reply README.md:39]: The git lfs currently doesn't work, as per #40 , where Ilaria has hit an error whilst trying to sync the files as we've exceeded are storage quote.. I think we probably want to fix this then before merging in these changes :smile:

**Review by hamishbeath — DISMISSED** [review]
all ok - would be good to have a conversation bout Git LFS and what this means for dependencies and maintaining the functionality of the code.

**Comment by phil-sandwell** [comment]
All of the changes look good to me. Seems like the final component to resolve is the LFS issue as per #40 and then this is good to go, all other comments are minor and/or inconsequential.

File: README.md (hunk 1 above)
Comment (BenWinchester) [reply README.md]: @phil-sandwell , this process has now been changed to use Zenodo, see updated REAMDE.MD file :smile:

**Comment by BenWinchester** [comment]
Re: Git LFS, these files have now been transferred over to Zenodo and we've got a DOI that I've put in the README :smile:
This closes #40 🙂 :

[DOI](https://doi.org/10.5281/zenodo.6647054)

**Review by hamishbeath — APPROVED** [review]

**Review by hamishbeath — APPROVED** [review]

**Review by hamishbeath — APPROVED** [review]

**Pull request merged**